hw4/02-sessions.js

```diff
@@ -1,16 +1,47 @@
 const express = require('express');
 const session = require('express-session');
 const app = express();
-const port = process.env.PORT || 5000;
+const port = process.env.PORT || 5001;
 
 // Add your code here
 
 // Use the express-session module
-app.use(/** ... */);
+app.use(
+  session({
+  store: new session.MemoryStore(),
+  secret: 'a secret to sign the cookie',
+  resave: false,
+  saveUninitialized: false,
+  cookie: {
+    maxAge: 86400000,
+  },
+})
+);
 
-app.get('/', (req, res) => {
+app.get('*', (req, res) => {
   // Add your code here
-});
+  res.status(200);
+  res.set({'Content-Type': 'text/plain'});
+  if (req.session.visit === undefined){
+    req.session.visit = [];
+    req.session.visit.push('/');
+    res.send(`Current route: /  \n Welcome First timer`);
```

Comment:
On the first visit the Welcome message needs to be like **Welcome to <FULL_URL>** as mentioned in the homework document. Can you add that here?

Reply:
Yes, I will go ahead and add that.